### ISSUE TITLE
Describe How to contribute

### ISSUE THREAD

**Issue by Zelldon** [camunda/camunda-cloud-helm#206]
I would like to have a guide which makes it clearer how to fix or add new features which also covers the tests. Like what kind of tests are expected and how they should be written. Furthermore I would like to add an why we do it like that.
